src/main/java/WordleGame.java

```diff
@@ -0,0 +1,51 @@
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class WordleGame {
+    public Enum[] resulColorArray = new Enum[5];
+
+    public Enum[] showResultColorArray(String inputString, String secretString) {
+
+        List<Character> characterList = initCharacterList(secretString);
+        validateInputString(inputString);
```

Comment:
move validation to the start of the method

Reply:
fixed